lib/ts/recipe/emailpassword/api/signin.ts

```diff
@@ -39,9 +41,20 @@ export default async function signInAPI(
         userContext
     );
 
+    let session = await Session.getSession(
+        options.req,
+        options.res,
+        {
+            sessionRequired: false,
+            overrideGlobalClaimValidators: () => [],
+        },
+        userContext
+    );
+
     let result = await apiImplementation.signInPOST({
         formFields,
         tenantId,
```

Comment:
tenantId should be fetched from the session if it exists.

Reply:
All APIs that are doing session verification!

Reply:
Done